packages/aws-cdk-lib/aws-route53/lib/health-check.ts

```diff
@@ -0,0 +1,670 @@
+   *
+   * @default - if the type is CALCULATED, the default value is number of child health checks. Otherwise, it is not configured.
+   */
+  readonly healthThreshold?: number;
+
+  /**
+   * The status of the health check when CloudWatch has insufficient data about the state of associated alarm.
+   *
+   * @default - if the type is CLOUDWATCH_METRIC, the default value is InsufficientDataHealthStatus.LAST_KNOWN_STATUS. Otherwise, it is not configured.
+   */
+  readonly insufficientDataHealthStatus?: InsufficientDataHealthStatusEnum;
+
+  /**
+   * Specify whether you want Amazon Route 53 to invert the status of a health check, so a health check that would normally be considered unhealthy is considered healthy, and vice versa.
+   *
+   * @default false
+   */
+  readonly inverted?: boolean;
+
+  /**
+   * The IPv4 or IPv6 IP address for the endpoint that you want Amazon Route 53 to perform health checks on. If you don't specify a value for IPAddress, Route 53 sends a DNS request to resolve the domain name that you specify in FullyQualifiedDomainName at the interval that you specify in RequestInterval. Using an IPv4 address that DNS returns, Route 53 then checks the health of the endpoint.
+   *
+   * @default - not configured
+   */
+  readonly ipAddress?: string;
+
+  /**
+   * Specify whether you want Amazon Route 53 to measure the latency between health checkers in multiple AWS regions and your endpoint, and to display CloudWatch latency graphs on the Health Checks page in the Route 53 console.
+   *
+   * @default - if the type is CALCULATED it's not configured
+   * - if the type is CLOUDWATCH_METRIC it's not configured
+   * - otherwise, the default value is false.
+   */
+  readonly measureLatency?: boolean;
+
+  /**
+   * The port on the endpoint that you want Amazon Route 53 to perform health checks on.
+   *
+   * @default - if the type is HTTP or HTTP_STR_MATCH, the default value is 80.
+   * - if the type is HTTPS or HTTPS_STR_MATCH, the default value is 443.
+   * - otherwise, it is not configured.
+   */
+  readonly port?: number;
+
+  /**
+   * An array of region identifiers that you want Amazon Route 53 health checkers to check the health of the endpoint from.
+   *
+   * Valid values are: 'us-east-1', 'us-west-1', 'us-west-2', 'eu-west-1', 'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'sa-east-1'
+   * @default - if the type is CALCULATED, CLOUDWATCH_METRIC, or RECOVERY_CONTROL, this property is not configured.
+   * - otherwise, the default value is all valid regions: 'us-east-1', 'us-west-1', 'us-west-2', 'eu-west-1', 'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'sa-east-1'
+   */
+  readonly regions?: ('us-east-1' | 'us-west-1' | 'us-west-2' | 'eu-west-1' | 'ap-southeast-1' | 'ap-southeast-2' | 'ap-northeast-1' | 'sa-east-1')[];
```

Comment:
I have the same maintainability concerns for this JSDoc and this type, I would just revert this type to a `string[]` and add a link to the Cfn documentation.

Just as an FYI, you could have used a `const` assertion to derive this type from the `VALID_REGIONS` array, see this [example Playground](https://www.typescriptlang.org/play/?#code/MYewdgzgLgBAagQQDIEkAiB9ASgUQOIoDyAcgMowC8MA2gOQCuEAtAKYCG0TAjLQDQwNmAdxaceAXRgcYoSFADcAKGUBLMFBYAnAGZtgLGAAVNIAA4QYAb0UxbMTSwDmK8BABcMABRQAnqZYg2vDI6Nj4RGTUYPQAtgBGWuIAlNTiSgC+yrLQMABubAA2KgAmHsZmFlSWDs6uHnSMrBxQ3LTi6UrZsGr5RaVGJuaUVjUukPWCTZwATG0dQA)